PocketKit/Sources/PocketKit/LoggedOut/LoggedOutViewController.swift

```diff
@@ -7,6 +7,11 @@ class LoggedOutViewController: UIHostingController<LoggedOutView> {
     convenience init(viewModel: LoggedOutViewModel) {
         self.init(rootView: LoggedOutView(viewModel: viewModel))
     }
+    
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        guard UIDevice.current.userInterfaceIdiom == .phone else { return .all }
```

Comment:
🤔 Would using the view controller's `traitCollection.userInterfaceIdiom` work here as well? I feel it may be better to check the idiom within the context of the current view controller over the device (we also won't have an implicit dependency on `UIDevice`).

Reply:
good point on the dependency, thanks!